### PR TITLE
Document bugs linked to lack of support for non-MediaStream values of srcObject

### PR DESCRIPTION
**Pull request by dontcallmedom**
Triggered by review of https://github.com/mdn/content/issues/1704

### CODE REVIEW

File: api/HTMLMediaElement.json [hunk 1]
```diff
@@ -3835,12 +3835,12 @@
             "samsunginternet_android": {
               "partial_implementation": true,
               "version_added": "6.0",
-              "notes": "Currently only supports <code>MediaStream</code> objects."
+              "notes": "Currently only supports <code>MediaStream</code> objects - see <a href='https://crbug.com/506273'>issue </a>."
```
Comment (ddbeck) [inline api/HTMLMediaElement.json:3838]:
```suggestion
              "notes": "Only supports <code>MediaStream</code> objects (see <a href='https://crbug.com/506273'>bug 506273</a>)."
```

File: api/HTMLMediaElement.json [hunk 2]
```diff
@@ -3835,12 +3835,12 @@
...
             },
             "webview_android": {
               "version_added": "52",
               "partial_implementation": true,
-              "notes": "Currently only supports <code>MediaStream</code> objects."
+              "notes": "Currently only supports <code>MediaStream</code> objects - see <a href='https://crbug.com/506273'>issue </a>."
```
Comment (ddbeck) [inline api/HTMLMediaElement.json:3843]:
```suggestion
              "notes": "Only supports <code>MediaStream</code> objects (see <a href='https://crbug.com/506273'>bug 506273</a>."
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thank you, @dontcallmedom. I've made a few suggestions to improve the style/consistency of the notes (getting rid of "currently" was convenient, since we're here now).

Also, I think it's very unlikely that Edge is unaffected. Can you copy the note from Chrome to Edge?

**Comment by dontcallmedom** [comment]
added the note to Edge as well

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**